docs/tutorials/geomesa-examples-authorizations.rst

```diff
@@ -55,7 +55,7 @@ Prerequisites
 
 Before you begin, you must have the following:
 
--  an instance of Accumulo 1.5 or 1.6 running on Hadoop 2.2.x
+-  an instance of Accumulo |accumulo_version| running on Hadoop |hadoop_version|
```

Comment:
We should at some point actually check all of the tutorials vs. 1.3.0-m0. I know some of them have been tested (Accumulo and Kafka) but maybe not the others. (Obviously beyond the scope of this PR!)